src/frontend/src/tests/lib/utils/onramper.utils.spec.ts

```diff
@@ -22,6 +35,10 @@ describe('buildOnramperLink', () => {
 				{ cryptoId: 'btc', wallet: 'bitcoin_wallet_address' },
 				{ cryptoId: 'icp', wallet: 'icp_wallet_address' }
 			],
+			networkWallets: [
```

Comment:
Can we use real addresses for the mocks? That would be a bit more readable regarding what's expected for me at least.